pyworkflow/mapper/sqlite_db.py

```diff
@@ -51,6 +51,7 @@ def _createConnection(self, dbName, timeout):
         if self._reuseConnections and dbName in self.OPEN_CONNECTIONS:
             self.connection = self.OPEN_CONNECTIONS[dbName]
         else:
+            self.closeConnection(dbName)  # Close the connect if exists for this db
```

Comment:
Can this be causing --> http://scipion-test.cnb.csic.es:9980/#/builders/88/builds/49/steps/4/logs/stdio